riscv-runtime/Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/powdr-labs/powdr"
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive", "rc"] }
 serde_cbor = { version = "0.11.2", default-features = false, features = ["alloc"] }
 powdr-riscv-syscalls = { path = "../riscv-syscalls" }
+tiny-keccak = { version = "2.0.2", features = ["keccak"] }
```

Comment:
I don't think this should be here. Why do we need this dependency here for tests?

Reply:
This is because currently the keccak.asm isn't ready and depends on Chris's circuit compiler #1218 to be merged first, so we put a placeholder keccak.asm. To test the Rust wrapper, we instead use the tiny-keccak.

Reply:
I'm removing this now and leaving the `keccakf` Rust wrapper as unimplemented.